content/en/Methodologies/mobile.md

```diff
@@ -98,13 +81,8 @@ When our pentesters discover a vulnerability, they use various techniques to mea
 
 Our pentesters use various privilege escalation methods to impersonate different users, possibly
```

Comment:
This is more in-line to our web and API testing methodology. The Mobile application is different and primarily focuses on data storage, privacy, platform permission, security controls, reverse engineering and anti tampering mechanism etc.